api/v1/dpuoperatorconfig_types.go

```diff
@@ -41,6 +41,7 @@ type DpuOperatorConfigStatus struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
+//+kubebuilder:resource:scope=Cluster
```

Comment:
This is all that is needed to change the scope to "cluster scoped"?

Reply:
yes

Reply:
Shouldn't the RBAC needs to also change?